WordPressAuthenticatorTests/Mocks/MockOnePasswordFacade.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+@testable import WordPressAuthenticator
+
+class MockOnePasswordFacade: OnePasswordResultsFetcher {
+    var error: OnePasswordError?
+    var username: String?
+    var password: String?
+    var otp: String?
+    
+    init(username: String, password: String, otp: String?) {
+        self.error = nil
+        self.username = username
+        self.password = password
+        if let otp = otp {
+            self.otp = otp
+        } else {
+            self.otp = nil
+        }
```

Comment:
Because both `otp` values are `String?`, you can just do `self.otp = opt`. If the input `otp` is `nil`, `self.otp` will be `nil`, too.